2024-08-09 05:32:51.675 INFO page_ 1:conftest.py:34: new page opened: [<Page url='about:blank'>]
2024-08-09 05:32:51.681 DEBUG tests_api.utils.logger:logger.py:23: > GET https://dev.agentum.pro/backend/api/user-answer {'method': 'GET', 'url': 'https://dev.agentum.pro/backend/api/user-answer', 'params': {'question_id': 1, 'answer': 'Some text'}, 'headers': {'Content-type': 'application/json', 'Authorization': 'Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz'}}
2024-08-09 05:32:51.682 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-08-09 05:32:51.795 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "GET /backend/api/user-answer?question_id=1&answer=Some+text HTTP/1.1" 502 157
2024-08-09 05:32:51.796 DEBUG tests_api.utils.logger:logger.py:27: < 502 https://dev.agentum.pro/backend/api/user-answer?question_id=1&answer=Some+text <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.18.0</center>
</body>
</html>

2024-08-09 05:32:51.796 DEBUG tests_api.utils.logger:logger.py:35: curl -X GET -H 'Accept: */*' -H 'Accept-Encoding: gzip, deflate' -H 'Authorization: Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz' -H 'Connection: keep-alive' -H 'Content-type: application/json' -H 'User-Agent: python-requests/2.31.0' 'https://dev.agentum.pro/backend/api/user-answer?question_id=1&answer=Some+text'